src/endpoint/cert_worker.js

```diff
@@ -0,0 +1,44 @@
+/* Copyright (C) 2023 NooBaa */
+'use strict';
+
+const dbg = require('../util/debug_module')(__filename);
+const config = require('../../config.js');
+const ssl_utils = require('../util/ssl_utils');
+
+/**
+ * Updates s3 https server(s) when certificate files are updated.
+ * Note ODF rotates certificates every 13 months.
+ * Some more details in operator's doc/ssl-dns-routing.md -
+ * https://github.com/noobaa/noobaa-operator/blob/master/doc/ssl-dns-routing.md
+ */
+class CertWorker {
+
+    /**
+     * @param {{
+     *   name: string;
+     *   https_server: https.Server;
+     *   sts_server: https.Server;
+     * }} params
+     */
+    constructor({ name, https_server, sts_server }) {
+        this.name = name;
+        this.https_server = https_server;
+        this.sts_server = sts_server;
+    }
+
+    async run_batch() {
+        dbg.log2('certificate bg worker: running check for certificate update');
```

Comment:
let keep the format we usually use: 
```suggestion
        dbg.log2('cert_worker: running check for certificate update');
```